OpenMEEG/tools/CMakeLists.txt

```diff
@@ -60,11 +71,11 @@ if (USE_VTK)
     add_executable(om_add_dataset_to_vtk add_dataset_to_vtk.cpp)
     target_link_libraries (om_add_dataset_to_vtk OpenMEEG ${VTK_LIBRARIES})
     install(TARGETS om_vtp_to_meshes om_meshes_to_vtp om_add_dataset_to_vtk DESTINATION bin)
+    if (BUILD_TESTING)
+        OPENMEEG_TEST(Tool-om_meshes_to_vtp ${CMAKE_CURRENT_BINARY_DIR}/om_meshes_to_vtp -i1 ${CMAKE_SOURCE_DIR}/data/Head1/cortex.1.tri -i2 ${CMAKE_SOURCE_DIR}/data/Head1/skull.1.tri -i3 ${CMAKE_SOURCE_DIR}/data/Head1/scalp.1.tri -n1 "cortex" -n2 "skull" -n3 "scalp" -o ${OpenMEEG_BINARY_DIR}/tests/Head1.vtp)
```

Comment:
actually we just needed another target name (not the same as the executable), to either add a prefix ("tool" as we did), or remove the prefix "om_" as done for other tests "assemble -h" for example.